CONTRIBUTING.md

```diff
@@ -1,22 +1,56 @@
 ## Contribution Guide
 
-Thank you for taking interest in contributing to Tracee!  
+Thank you for taking interest in contributing to Tracee! This document covers our working practices and conventions.
 
 ## Issues and Discussions
 
-- Feel free to open issues for any reason as long as you make it clear what this issue is about: bug/feature/proposal/comment.
-- For questions and general discussions, please do not open an issue, and instead create a discussion in the "Discussions" tab.
-- Please spend a minimal amount of time giving due diligence to existing issues or discussions. Your topic might be a duplicate. If it is, please add your comment to the existing one.
-- Please give your issue or discussion a meaningful title that will be clear for future users.
-- The issue should clearly explain the reason for opening, the proposal if you have any, and any relevant technical information.
-- For technical questions, please explain in detail what you were trying to do, provide an error message if applicable, and your versions of Tracee and your environment.
+We encourage open discussion and collaboration using both GitHub Issues and Discussions.  
+
+- [Discussions](https://github.com/aquasecurity/tracee/discussions) are free-style conversational tool, we use them for conversations.
+- [Issues](https://github.com/aquasecurity/tracee/issues) are project management tool, we use them to keep track on who's working on what and plan ahead.
+
+If you have a suggestion, question, or a general comment - please use Discussions. If there's a clear work item (including bugs) - you can open an Issue.
+
+### Discussions:
+
+1. We have the following discussion topics: 
+  1. [Announcements](https://github.com/aquasecurity/tracee/discussions/categories/announcements): One way communication from the team to the community. Consider this like our mini blog
+  2. [Questions and Help](https://github.com/aquasecurity/tracee/discussions/categories/questions-and-help): For help and support. Consider this similar to StackOverflow.
+  3. [Development](https://github.com/aquasecurity/tracee/discussions/categories/development): For discussing potential features, and collaborating on their design.
+
+### Issues:
+
+1. Every issue needs to be actionable and assignable. Consider the scope of the issue if assigned to one person, and break down if necessary.
+1. Be clear and definitive when composing issues. For bug reports, include detailed error messages and environment description. For features, include a clear scope and acceptance criteria.
```

Comment:
Should we add an example? People might not be familiar with the terms scope or acceptance criteria. This seems simple enough to be one https://github.com/aquasecurity/tracee/issues/450